demo/responsive-demo.js

```diff
@@ -30,18 +31,18 @@ <h3>Small Table</h3>
 			</tbody>
 		</table></d2l-table-wrapper>
 		<h3>Simple Table</h3>
-		<d2l-table-wrapper><table class="d2l-table" selectable>
-			<tr>
+		<d2l-table-wrapper><table class="d2l-table" selectable="">
+			<tbody><tr>
```

Comment:
Interesting... Looks like polymer-modulizer's parser/serializer adds tbody. Makes sense considering the HTML Table spec.